docs/source/el7toel8/envars.md

```diff
@@ -79,6 +80,39 @@ case they are changed. However, in some cases it's not wanted and it leads
 in malfunction network configuration (e.g. in case the bonding is configured
 on the system). It's expected that NICs have to be handled manually if needed.
 
+## LEAPP_DATABASE_FORCE_SYNC_ON
+
+If set to 1, leapp will explicitly enable synchronization on the SQLite database.
+Enabling the synchronization has negative impact on the performance
+(sometimes very negative) however it is more reliable in case of extreme
+situations (e.g. lost power).
+Note the synchronization is nowadays switched off by default only during the phases
+executed before the reboot of the system to the upgrade environment which we consider
+safe, so we do not expect that someone would want to use this option now.
+However, we are testing and discussing the possibility to switch off
```

Comment:
Are our future plans necessary for customers to know about?

Reply:
@mportman12  I am not looking at it as an information shared with customers, but an information shared with the community (which could include customers if they read it). From that point it should be ok to mention it. It's probably not necessary, but I felt it as a good thing to add - it's possible that same good decision could be to drop it, as it has possibly low value :D  I am still thinking whether actually to document the envar itself or drop it completely.

Reply:
@pirat89 Ok, in that case recommend the following rewrite. The current text is made up of very long sentences, so it's hard to follow the train of thought:
"Note the synchronization is currently disabled by default only during the phases
executed before the reboot of the system to the upgrade environment. Because this is a safe phase to disable synchronization, Red Hat does not anticipate many users will want to re-enable synchronization. 
However, Red Hat is currently testing and discussing the possibility of disabling the synchronization for all phases of the upgrade. If synchronization becomes disabled by default for the entire upgrade, users can use this environment variable for troubleshooting purposes."

Reply:
@mportman12  I like this, just I will exchange here Red Hat by "we" in the upstream documentation. But it sounds better now. I will update it during the start of the week. thanks!!

Reply:
dropping the part about the future plans.